src/views/InvolvementProfile/components/Membership/components/MemberList/components/MemberListItem/index.js

```diff
@@ -49,6 +52,7 @@ const MemberListItem = ({
   onToggleIsAdmin,
 }) => {
   const [isEditDialogOpen, setIsEditDialogOpen] = useState(false);
+  const [titleDialog, setTitleDialog] = useState(member.Description);
```

Comment:
You added this new state variable but also kept the original `title` state below. Is there a reason to have both? You seem to mix them in some places, would could cause some bugs, and results in repetitive code.